sdk/workloadmonitor/Microsoft.Azure.Management.WorkloadMonitor/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Need to follow the directions here:  https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-directory-structure-and-sdk-generation for generating the SDK.  This will also generate a file that records the branch and commit in the rest-api-specs repo, which needs to be checked in as part of this PR

Reply:
I think I followed these steps and added the two filed that were auto-generated